index.html

```diff
@@ -8,11 +8,16 @@
   <link rel="stylesheet"
   href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
-<body>
-  <header>
-    <nav class="header">
+<body class="container">
+  <header class="head">
+    <nav class="header hide-desktop">
       <img src="assets/images/menu.png" class="menu-btn"">
```

Comment:
- [ ] You did great with an attempted fix to the hamburger, 👏 . However, kindly align the margin around the borders as its spaced on the Figma design.
- This will ensure your work has a better user experience as it is on the Figma design 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50990925/236510835-2624e528-cff3-4324-9674-b142bb2eb7a6.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/236510932-38eee4be-5fa6-41d6-8cc4-59c0ecc9b904.png)